src/components/ProcessedErrorSummary.js

```diff
@@ -17,13 +17,14 @@ const ProcessedErrorSummary = (props) => {
         heading={props.t("There is a problem")}
         onHandleErrorClick={props.onHandleErrorClick}
         errors={errors}
+        role="alert"
       />
```

Comment:
Are there changes to be made to the ErrorSummary govuk-react component too?
e.g. tabindex, aria label, href